OpenOversight/tests/conftest.py

```diff
@@ -236,6 +236,12 @@ def teardown():
         connection.close()
         session.remove()
 
+        # Since Flask-Login now records the logged in user on the Flask `g` object and we persist the app context
+        # (in the app fixture) for the entire testing session, we need to reset this value manually.
+        #
+        # Relevant PR: https://github.com/maxcountryman/flask-login/pull/691
+        g.pop("_login_user", None)
```

Comment:
Just a note, let's merge #167 before this and drop this commit so we can keep them as separate changes!